handlers/main.yml

```diff
@@ -1,14 +1,38 @@
 ---
 
 - name: reload systemd
-  systemd: daemon_reload=yes
+  systemd:
+    daemon_reload: yes
   when: ansible_distribution_release in ['bionic'] and ansible_service_mgr == 'systemd'
 
 - name: reload supervisor
   command: supervisorctl reload
 
 - name: restart nginx
-  service: name=nginx state=restarted
+  service:
+    name: nginx
+    state: restarted
+
+- name: start mysql
+  service:
+    name: mysql
+    state: started
+
+- name: start postgresql
+  service:
+    name: postgresql
+    state: started
```

Comment:
why these handlers? Database do not need to be restarted since we do not touch their configurations, moreover, as I already stressed, the database management should be the concern of another role.

I understand you want to try to make it easier for the everyday user, but the everyday user who doesn't want to invest time in doing an advanced OpenWISP deployment should stick to sqlite, otherwise we'll end up having to give support to casual users in how to configure and manage their RDBMS which we surely don't have the resources to do.